api/v1beta1/loadbalancer_types.go

```diff
@@ -182,33 +182,65 @@ type LoadBalancerInfrastructure struct {
 type OpenstackImageRef struct {
 	// ImageID is the image ID used for requesting virtual machines.
 	// +optional
-	ImageID *string `json:"image_id,omitempty"`
+	ImageID *string `json:"imageID,omitempty"`
 	// ImageName is the name of the image used for requesting virtual machines.
 	// ImageName is only used if ImageID is not defined.
 	// +optional
-	ImageName *string `json:"image_name,omitempty"`
+	ImageName *string `json:"imageName,omitempty"`
 	// ImageSearch is a search string to find the image used for requesting virtual machines.
 	// Search will be performed in metadata of the images.
 	// ImageSearch is only used if ImageName and ImageID are not defined.
 	// +optional
-	ImageSearch *string `json:"image_search,omitempty"`
+	ImageSearch *string `json:"imageSearch,omitempty"`
+
+	// Deprecated: used for migration
```

Comment:
This will be rendered in the crd so it makes sense to reference to the new field like
```suggestion
	// Deprecated: use imageID instead
```